hphp/runtime/vm/jit/retranslate-all.md

```diff
@@ -0,0 +1,108 @@
+Overview
+--------
```

Comment:
This is fantastic, but it should be moved to hphp/doc/hackers-guide, and could use some more detail about the ARM side of things (and also looks like it's somewhat out of date).